x-pack/test/functional/apps/apm/feature_controls/apm_security.ts

```diff
@@ -64,6 +64,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         const navLinks = await appsMenu.readLinks();
         expect(navLinks.map((link) => link.text)).to.eql([
           'Overview',
+          'Alerts',
```

Comment:
Is this about the Observability menu? Couldn't we add `Cases` here as well?
<img width="236" alt="Screenshot 2021-08-19 at 13 35 59" src="https://user-images.githubusercontent.com/2852703/130062087-0ba49e7e-41d8-4d90-ab34-2a51cd807938.png">

Reply:
Given the role definition used for the test I would expect cases not to be shown.